.github/pull_request_template.md

```diff
@@ -1,35 +1,35 @@
 ## Proposed changes
 
-Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
+Describe the changes here.
 
-## Fixes
+## Issues
 
-If it fixes a bug or resolves a feature request, be sure to link to that issue.
+Links to any issues resolved.
 
 ## Types of changes
 
 What types of changes does your code introduce to agents-aea?
 _Put an `x` in the boxes that apply_
 
-- [ ] Bugfix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Bugfix (non-breaking change that fixes an issue)
+- [ ] New feature (non-breaking change that adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to stop working as expected)
+- [ ] Something else (e.g. test, package, script, example, deployment, infrastructure, ...)
 
 ## Checklist
 
 _Put an `x` in the boxes that apply._
 
-- [ ] I have read the [CONTRIBUTING](https://github.com/fetchai/agents-aea/blob/main/CONTRIBUTING.md) doc
-- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
-- [ ] Lint and unit tests pass locally with my changes and CI passes too
-- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have based my branch, and I am making a pull request against, the `develop` branch.
+- [ ] Lint and unit tests pass locally with my changes.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
```

Comment:
How about adding an "If applicable" section for the remaining items? Adding tests is often not necessary and it would be nice to have at least one portion of the checklist that should really have all items checked before creating the PR.

```suggestion
### If applicable

- [ ] I have added tests that prove my fix is effective or that my feature works.
```